### DISCUSSION TITLE
Tsys from the continuous calibration

### DISCUSSION THREAD

**Discussion by vlbi-jun-yang** [nvi-inc/fs#155] [Q&A] [answered]
In case of the 80 Hz continuous calibration, it seems that the FS Tsys estimate excludes the contribution of the noise sources.  Is my understanding correct?

**Comment by wehimwich** [comment, marked as answer]
For DBBC(2) BBCs, I think so. The effect is that Tsys is underestimated by about half of the Tcal value. Said another way, if the Tcal is 10% of Tsys, then the measured Tsys will be low by about 5%. Clearly this is wrong. This dates back to the initial implementation of continuous cal for DBBC(2) in 2012. All the implications of continuous cal may not have been clear at that time. A 10% Tcal may be unusually high.

The (generally not useful) DBBC3 continuous calibration Tsys measured without multicast also suffers from this. The coming multicast support for the DBBC3 does not have this issue, nor does RDBE support. Continuous cal for VLBA racks was never supported.

Thank you for finding this problem. I am sorry it exists. I am curious how you found it: reading the code, verifying log output, issues with calibration data downstream, or something else.

This should be easy to fix. However, we need a test bed to verify the change.

**Reply by vlbi-jun-yang** [reply]
Thanks for the confirmation. I notice this by verifying the log output. This is also related to the SEFD observations. It is better also to pop out a message to clarify this in case of continuous calibration. Sure, you are welcome to use the Onsala VLBI backends to verify the change. The EVN session will end on Nov 11. After that, we should have one VLBI backend free for the verification.

**Reply by wehimwich** [reply]
Good work on verifying this from the log! I am glad you were interested and able to look into this. I am going to convert this to an issue. I will contact you directly to coordinate on test time. Thanks.
